lib/hanami/middleware/body_parser/class_interface.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'errors'
+require_relative 'parser'
+
+module Hanami
+  module Middleware
+    class BodyParser
+      module ClassInterface
+        # @since x.x.x
+        # @api private
+        def for(parser)
+          if parser_name?(parser)
+            require_parser(parser)
+          elsif parser_class?(parser)
+            parser.new
+          elsif parser_instance?(parser)
+            parser
+          else
+            raise UnknownParserError.new(parser)
+          end
+        end
+
+        private
+
+        # @since 1.3.0
```

Comment:
And since this method is now moved, should probably change this `@since` doc tag? Since these are tagged as private we're OK to move them around, right?